gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -190,15 +195,24 @@ public TransportChannel getTransportChannel() throws IOException {
   private TransportChannel createChannel() throws IOException {
     ManagedChannel outerChannel;
```

Comment:
This variable is first used only on line 211. Please move this declaration lower (on line 210), so declaration is next to first usage, which brings it in accordance to best practices (declaration and first usage of local variables should close to each other).